app/controllers/api/lessons_controller.rb

```diff
@@ -16,11 +16,12 @@ def create
   def update
     lesson = Lesson.find params[:id]
     # TODO: If the update only changes price and not time, then we shouldnt' send a reschedule email, but a Lesson Price Changed email.
-    if params[:is_paid]
-      # Send payment email
-    elif params[:start_time] and params[:end_time]
```

Comment:
@echang9 i realize why this wasn't working before. It's suppsed to be `elsif`, not `elif`. Do you think the order should be reverted to what it was before?